layout/plan.c

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2012-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
This is a new file. So, I propose to have:
```
* Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

Also, please add the following at the beginning of this file:
```
/* -*- C -*- */
```

Reply:
Fixed.